### PR TITLE
INT-4102 fixes for deployment

### PR DESCRIPTION
**Pull request by jflinchbaugh**
First step: catch up small fixes to ensure the operator deploys that we learned with IQ Operator.

JIRA: https://issues.sonatype.org/browse/INT-4102 (part 1)
Jenkins: https://jenkins.ci.sonatype.dev/view/all/job/integrations/job/cloud/job/operator-nxrm3/job/feature-snapshots/job/INT-4102-release-latest/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -39,14 +39,6 @@ node('ubuntu-zion') {
...
     }
   }
-
-  stage('Build') {
-    // OsTools.runSafe(this, 'scripts/bundle.sh')
-  }
-
-  stage('Archive') {
-      archiveArtifacts artifacts: archiveName, onlyIfSuccessful: true
-  }
```
Comment (jflinchbaugh) [inline Jenkinsfile:49]: these won't work for now until the other jenkins story is done.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ and push your operator image to quay.io to make it available for installation.
    for a local Openshift 4 environment.
 2. Ensure you have a personal quay.io account.
 3. Build and deploy the operator image to your personal quay.io repository:
-   1. `operator-sdk build quay.io/<username>/nxrm-operator-certified:[operator-version]`
+   1. `docker build . -f build/Dockerfile --tag quay.io/<username>/nxrm-operator-certified:[operator-version]`
```
Comment (jflinchbaugh) [inline README.md:14]: newer versions of operator-sdk no longer do this build, so just use docker.

File: scripts/templates/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # {{templateWarning}}
-FROM quay.io/operator-framework/helm-operator:v0.16.0
+FROM quay.io/operator-framework/helm-operator:v1.5.0
```
Comment (jflinchbaugh) [inline scripts/templates/Dockerfile:2]: upgrade to latest base image.

File: README.md (hunk 2 above)
Comment (collinpeters) [reply README.md:14]: what does "no longer do this build" mean?

**Review by collinpeters — APPROVED** [review]
+1

File: README.md (hunk 2 above)
Comment (jflinchbaugh) [reply README.md:14]: operator-sdk no longer has a "build" command, so we revert to docker to do it.

**Comment by jflinchbaugh** [comment]
merging to give it a shot against red hat's services.

**Pull request merged**